packages/flutter/lib/src/widgets/async.dart

```diff
@@ -300,7 +300,7 @@ class AsyncSnapshot<T extends Object> {
 ///    itself based on a snapshot from interacting with a [Stream].
 ///  * [FutureBuilder], which delegates to an [AsyncWidgetBuilder] to build
 ///    itself based on a snapshot from interacting with a [Future].
-typedef AsyncWidgetBuilder<T extends Object> = Widget Function(BuildContext context, AsyncSnapshot<T> snapshot);
```

Comment:
Wouldn't we also have to remove this on the various widgets below to make https://gist.github.com/goderbauer/244c5f19f69bb36d4a136d176ad189c4 work?

Reply:
Oh, no, that's not required. Never mind.